Objects/typeobject.c

```diff
@@ -3782,6 +3782,8 @@ object_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
         PyObject *joined;
         PyObject *comma;
         _Py_static_string(comma_id, ", ");
+        _Py_IDENTIFIER(sorted);
```

Comment:
Why has this `_Py_IDENTIFIER(sorted);` been added?

Reply:
I failed at resolving merge conflicts.